matrix-sdk/src/main/java/org/matrix/androidsdk/rest/model/TaggedEvents.kt

```diff
@@ -75,7 +76,7 @@ data class TaggedEventInfo(@JvmField
                            @JvmField
                            @SerializedName("tagged_at")
                            var taggedAt: Long? = null
-) {
+) : Serializable {
```

Comment:
Maybe `TaggedEventsContent` (in the same file)  should also implement `Serializable`?

Reply:
Ok, this is not required for the moment, but this may be useful in the future. I will update this

Reply:
done